src/main/java/org/broadinstitute/hellbender/tools/gvs/common/SchemaUtils.java

```diff
@@ -83,6 +83,17 @@ public static long encodeLocation(String chrom, int position) {
         return (long) chromosomeIndex * chromAdjustment + (long) position;
     }
 
+
+    public static long encodeCompressedRefBlock(String chrom, long position, long length, long stateAsInt) {
+        long chromosomeIndex = ChromosomeEnum.valueOfContig(chrom).index;
+        long packedBytes = ((0xFFFF & chromosomeIndex) << 48) |
+                ((0xFFFFFFFF & position) << 16) |
+                ((0xFFF & length) << 4) |
```

Comment:
are we very sure there are no indels longer than this?

Reply:
This length does not refer to the length of a variant, but the length of a _reference block_.  These are capped at 1k in our code, and the number of bits allocated would allow us to expand that up to 4k without having to adjust the representation